backend/api/routers/host_router.py

```
@@ -0,0 +1,245 @@
+from fastapi import APIRouter, HTTPException
+from typing import Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)